src/services/DataSourceDealsTypes.ts

```diff
@@ -4,6 +4,7 @@ import { IDealDiscussion } from "entities/DealDiscussions";
 import { IDealRegistrationTokenSwap } from "entities/DealRegistrationTokenSwap";
 import { IDealTokenSwapDocument } from "./../entities/IDealTypes";
 import { Address } from "./EthereumService";
+import { IDocumentUpdates } from "./FirestoreTypes";
```

Comment:
`DataSourceDealsTypes.ts` is supposed to be a generic interface.
Is there another way to specify this type? (and not add a dependency on firestore)

Reply:
agreed

Reply:
IDocumentUpdates  should be defined here, not in FirestoreTypes